### PR TITLE
(BOLT-1302) Document installation for puppet-bolt on fedora 30

### PR DESCRIPTION
**Pull request by donoghuc**
This commit adds the installation instructions for installing bolt from the puppet 6 collection.

### CODE REVIEW

**Comment by donoghuc** [comment]
Need to coordinate publishing this when we ship bolt packages for fedora 30.

**Comment by lucywyman** [comment]
Pinging @melamos

File: pre-docs/bolt_installing.md [hunk 1]
```diff
@@ -225,6 +225,13 @@ The Puppet repository for the YUM package management system is [http://yum.puppe
         sudo dnf install puppet-bolt
         ```
 
+    -   Fedora 30
+
+        ```
+        sudo rpm -Uvh https://yum.puppet.com/puppet6/puppet6-release-fedora-30.noarch.rpm
```
Comment (beechtom) [inline pre-docs/bolt_installing.md]:
```suggestion
        sudo rpm -Uvh https://yum.puppet.com/puppet6-release-fedora-30.noarch.rpm
```

Comment (donoghuc) [reply pre-docs/bolt_installing.md]: Not sure if that is intentionally different than for the other fedora platforms. Lets confirm with RE before we update. Hopefully there is a standard location to point to for all the fedora platform versions.

Comment (beechtom) [reply pre-docs/bolt_installing.md]: This is the standard location per RE. All release RPMs are in the top level directory.

**Comment by beechtom** [comment]
All other paths in the docs are correct.

**Comment by donoghuc** [comment]
@MFredette Somehow the pre-docs did not match the puppet docs. So I originally copied and pasted what we had for fedora 28 and 29 for fedora 30. The public website already has the correct info for all the yum packages. @beechtom fixed the pre-docs to match the public site, but the fedora 30 installation instructions will need to be updated.

**Comment by MFredette** [comment]
Rebuilding now.

**Pull request merged**